	In TeX Live we treat all output files as binary.

diff -ur -x Makefile.in -x aclocal.m4 -x autoconf.h.in -x configure lcdf-typetools-2.100.orig/liblcdf/filename.cc lcdf-typetools-2.100/liblcdf/filename.cc
--- lcdf-typetools-2.100.orig/liblcdf/filename.cc	2013-08-13 22:26:12.000000000 +0200
+++ lcdf-typetools-2.100/liblcdf/filename.cc	2014-01-09 12:47:42.000000000 +0100
@@ -126,5 +126,5 @@
     if (_actual || !_path)
 	return _actual;
     else
-	return fopen(_path.c_str(), binary ? "wb" : "w");
+	return fopen(_path.c_str(), "wb");
 }
diff -ur -x Makefile.in -x aclocal.m4 -x autoconf.h.in -x configure lcdf-typetools-2.100.orig/otftotfm/automatic.cc lcdf-typetools-2.100/otftotfm/automatic.cc
--- lcdf-typetools-2.100.orig/otftotfm/automatic.cc	2014-01-08 17:23:16.000000000 +0100
+++ lcdf-typetools-2.100/otftotfm/automatic.cc	2014-01-09 12:47:42.000000000 +0100
@@ -311,7 +311,7 @@
     String ls_r = writable_texdir + "ls-R";
     bool success = false;
     if (access(ls_r.c_str(), R_OK) >= 0) // make sure it already exists
-	if (FILE *f = fopen(ls_r.c_str(), "a")) {
+	if (FILE *f = fopen(ls_r.c_str(), "ab")) {
 	    fprintf(f, "./%s:\n%s\n", directory.c_str(), file.c_str());
 	    success = true;
 	    fclose(f);
@@ -675,7 +675,7 @@
 #endif
             {
                 fclose(f);
-                f = fopen(map_file.c_str(), "w");
+                f = fopen(map_file.c_str(), "wb");
                 fd = fileno(f);
             }
 
diff -ur -x Makefile.in -x aclocal.m4 -x autoconf.h.in -x configure lcdf-typetools-2.100.orig/otftotfm/otftotfm.cc lcdf-typetools-2.100/otftotfm/otftotfm.cc
--- lcdf-typetools-2.100.orig/otftotfm/otftotfm.cc	2013-08-13 22:26:12.000000000 +0200
+++ lcdf-typetools-2.100/otftotfm/otftotfm.cc	2014-01-09 12:47:42.000000000 +0100
@@ -555,7 +555,7 @@
 
     if (verbose)
 	errh->message("creating %s", filename.c_str());
-    FILE *f = fopen(filename.c_str(), "w");
+    FILE *f = fopen(filename.c_str(), "wb");
     if (!f) {
 	errh->error("%s: %s", filename.c_str(), strerror(errno));
 	return;
@@ -997,7 +997,7 @@
 #endif
     {
 	fclose(f);
-	f = fopen(filename.c_str(), "w");
+	f = fopen(filename.c_str(), "wb");
 	fd = fileno(f);
     }
 
diff -ur -x Makefile.in -x aclocal.m4 -x autoconf.h.in -x configure lcdf-typetools-2.100.orig/t1dotlessj/t1dotlessj.cc lcdf-typetools-2.100/t1dotlessj/t1dotlessj.cc
--- lcdf-typetools-2.100.orig/t1dotlessj/t1dotlessj.cc	2013-08-13 22:26:12.000000000 +0200
+++ lcdf-typetools-2.100/t1dotlessj/t1dotlessj.cc	2014-01-09 12:47:42.000000000 +0100
@@ -403,10 +403,10 @@
     // write it to output
     if (!outputf)
 	outputf = stdout;
-    if (binary) {
 #if defined(_MSDOS) || defined(_WIN32)
-	_setmode(_fileno(outputf), _O_BINARY);
+    _setmode(_fileno(outputf), _O_BINARY);
 #endif
+    if (binary) {
 	Type1PFBWriter w(outputf);
 	dotless_font->write(w);
     } else {
diff -ur -x Makefile.in -x aclocal.m4 -x autoconf.h.in -x configure lcdf-typetools-2.100.orig/t1rawafm/t1rawafm.cc lcdf-typetools-2.100/t1rawafm/t1rawafm.cc
--- lcdf-typetools-2.100.orig/t1rawafm/t1rawafm.cc	2013-08-13 22:26:12.000000000 +0200
+++ lcdf-typetools-2.100/t1rawafm/t1rawafm.cc	2014-01-09 12:47:42.000000000 +0100
@@ -352,6 +352,9 @@
 	if (!outf)
 	    errh->fatal("%s: %s", output_file, strerror(errno));
     }
+#if defined(_MSDOS) || defined(_WIN32)
+    _setmode(_fileno(outf), _O_BINARY);
+#endif
 
     write_afm(outf, font);
 
diff -ur -x Makefile.in -x aclocal.m4 -x autoconf.h.in -x configure lcdf-typetools-2.100.orig/t1reencode/t1reencode.cc lcdf-typetools-2.100/t1reencode/t1reencode.cc
--- lcdf-typetools-2.100.orig/t1reencode/t1reencode.cc	2013-08-13 22:26:12.000000000 +0200
+++ lcdf-typetools-2.100/t1reencode/t1reencode.cc	2014-01-09 12:47:42.000000000 +0100
@@ -1094,10 +1094,10 @@
 	if (!outf)
 	    errh->fatal("%s: %s", output_file, strerror(errno));
     }
-    if (binary) {
 #if defined(_MSDOS) || defined(_WIN32)
-	_setmode(_fileno(outf), _O_BINARY);
+    _setmode(_fileno(outf), _O_BINARY);
 #endif
+    if (binary) {
 	Type1PFBWriter w(outf);
 	font->write(w);
     } else {
diff -ur -x Makefile.in -x aclocal.m4 -x autoconf.h.in -x configure lcdf-typetools-2.100.orig/t1testpage/t1testpage.cc lcdf-typetools-2.100/t1testpage/t1testpage.cc
--- lcdf-typetools-2.100.orig/t1testpage/t1testpage.cc	2013-08-13 22:26:12.000000000 +0200
+++ lcdf-typetools-2.100/t1testpage/t1testpage.cc	2014-01-09 12:47:42.000000000 +0100
@@ -665,6 +665,9 @@
 	if (!outf)
 	    errh->fatal("%s: %s", output_file, strerror(errno));
     }
+#if defined(_MSDOS) || defined(_WIN32)
+    _setmode(_fileno(outf), _O_BINARY);
+#endif
 
     //font->undo_synthetic();
 
